packages/mlgmpidl/mlgmpidl.1.2.9/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+authors: ["Bertrand Jeannet" "Nicolas Berthier"]
+maintainer: "Nicolas Berthier <m@nberth.space>"
+dev-repo: "git+https://github.com/nberth/mlgmpidl.git"
+bug-reports: "https://github.com/nberth/mlgmpidl/issues"
+homepage: "https://www.inrialpes.fr/pop-art/people/bjeannet/mlxxxidl-forge/mlgmpidl/"
+license: "LGPL-2.1 with static linking exception"
+build: [
+  ["sh" "./configure" "--absolute-dylibs" {os = "macos"}]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
```

Comment:
The remove section is no longer needed in opam2 when you are just removing what you have installed. This does not prevent the merge though

Reply:
Interesting… just out of curiosity, how is "what you have installed" recorded?

Reply:
Opam2 tracks the files that are installed